.github/workflows/solr-synonyms-api-cd.yml

```diff
@@ -18,6 +18,10 @@ on:
         - sandbox
         - prod
 
+permissions:
```

Comment:
Remove this - [Invalid workflow file: .github/workflows/solr-synonyms-api-cd.yml#L26](https://github.com/bcgov/namex/actions/runs/16533120116/workflow)
The workflow is not valid. .github/workflows/solr-synonyms-api-cd.yml (Line: 26, Col: 3): Error calling workflow 'bcgov/bcregistry-sre/.github/workflows/backend-cd.yaml@main'. The nested job 'setup' is requesting 'contents: write', but is only allowed 'contents: read'. .github/workflows/solr-synonyms-api-cd.yml (Line: 26, Col: 3): Error calling workflow 'bcgov/bcregistry-sre/.github/workflows/backend-cd.yaml@main'. The nested job 'deploy' is requesting 'contents: write', but is only allowed 'contents: read'.